### ISSUE TITLE
How to reproduce the experimental data in the paper?

### ISSUE THREAD

**Issue by HHeracles** [MhLiao/MaskTextSpotterV3#20]
Hello @MhLiao .I used the downloaded model “trained_model.pth”，and set the MIN_SIZE_TEST parameter to the corresponding value，1000 or 1440.However,.The data obtained on data set Total_text( MIN_SIZE_TEST=1440) are respectively "hmean": 0.831966(strong),0.779740(weak),0.733809(generic).But the experimental data in your paper are respectively 83.3%,78.1%,74.2%(in table 3). And The data obtained on data set rotated_ic13_test_45( MIN_SIZE_TEST=1000) are respectively "precision": 0.8811274509803921, "recall": 0.6657407407407407, "hmean": 0.7584388185654007,.But the experimental data in your paper are respectively 88.5%,66.8.1%,76.2%(in table 1).  
All in all, the metrics running in the program are lower than the data provided in your paper.If we want to run the program to reproduce metrics in your paper, what changes should we make? Do We need to retrain the model?

**Comment by MhLiao** [comment]
@HHeracles Did you use the suggested thresholds for different types of the lexicon in the evaluation scripts?  The results you show are slightly lower, which may be caused by the thresholds.
By the way, what you mean should be the ICDAR 2015 dataset instead of Total-Text.

**Comment by HHeracles** [comment]
@MhLiao  Thanks for your reminding.But when I used the suggested thresholds( score_rec_seq = 0.9 or 0.8 or 0.7) for different types of the lexicon in the evaluation scripts, the results from the program are still about 0.2 percent lower than the data in the paper on Total_text or icdar2015, and on dataset RoIC13(angle45,or angle60), the results generated by the program are about 0.5% lower than the data in the paper. Maybe that's acceptable.

**Comment by nguyennm1024** [comment]
@MhLiao I got the same problem with @HHeracles. When I try to run your checkpoint in ICDAR 15 dataset, the result for the strong dictionary is "precision": 0.9221014492753623, "recall": 0.7351949927780452, "hmean": 0.8181087597106885. It much lower than the result in your paper is 83.3 for hmean. Should I retrain the model with ICDAR 15 dataset? I use your recommended threshold (score_det: 0.01 score_rec: 0.4 score_rec_seq: 0.7 lexicon_type: 3)

**Comment by thcheung** [comment]
@nguyennm1024 
My result is exactly the same as you: "hmean": 0.8181087597106885 with the dataset ICDAR 2015.

**Comment by HHeracles** [comment]
@MhLiao Hello！We used 4 Gpus, and the parameter was set as NUM_WORKES:4, the BASE_LR:0.01, and the other parameters are default values to train the model.But we cannot get the data performance you provided in the paper by using our own trained model weights.If we want to use four GPU2080 training models to achieve the same data performance as in the paper, do you have any suggestions for parameter setting or other tricks?

**Comment by MhLiao** [comment]
@HHeracles Have you followed the pretraining and finetuning schedules as suggested in the paper? In my opinion, you may need more iterations and a smaller initial learning rate if your batch size is smaller.
